.github/workflows/build-deb.yaml

```diff
@@ -0,0 +1,117 @@
+name: Build debian packages
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+
+env:
+  DEBIAN_FRONTEND: noninteractive
+  DEBCONF_NONINTERACTIVE_SEEN: true
+  TERM: dumb
+
+jobs:
+  build-deb-package:
+    name: Build ubuntu package
+    runs-on: ubuntu-latest
+
+    env:
+      AS_USER: runuser -u builder --
+
+    container:
+      image: ubuntu:devel
+
+    steps:
+    - name: Prepare container
+      run: |
+        echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90aptyes
+        apt update
+        apt upgrade
+        apt install git
+        git config --system --add safe.directory $PWD
+
+    - name: Checkout authd code
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        apt install equivs devscripts lintian
+        mk-build-deps -ir
```

Comment:
This is an old way of instlaling build deps, let’s not create the dummy package are really on `build-dep .` funcitonality of apt.